### PR TITLE
feat(consultation-portal): readme files

### PR DESCRIPTION
**Pull request by korsbacka**
# update readme file

https://veflausnir.atlassian.net/browse/KAM-1261

## What

- Update readme file in frontend
- Add readme file in api domain
- Add readme file in api client

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/consultation-portal/README.md [hunk 1]
```diff
@@ -1,27 +1,79 @@
-# Samráðsgáttin
+# Consultation Portal
 
 ## About
 
-TODO
+The aim of the consultation portal is to increase transparency and the possibility for the public and interested parties to participate in policy-making, regulstion and decision-making by public bodies.
+
```
Comment (diljath) [inline apps/consultation-portal/README.md:6]: stafsetningavilla: regulations

File: apps/consultation-portal/README.md [hunk 2]
```diff
@@ -1,27 +1,79 @@
...
+In the consultation portal, you can find plans for legislation, drafts of bills and regulations, documents on strategic planning (eg draft policies) and more. You can submit a comment or suggestion, and it is also possible to subscribe to automatic monitoring of information, whether by subject area, organization or specific issue.
+
+To be able to submit a comment or suggestion and subscribe to automatic monitoring, the user needs to use his "Rafræn skilríki" credentials.
```
Comment (diljath) [inline apps/consultation-portal/README.md:10]: Rafræn skilríki > electronic ID ?

File: apps/consultation-portal/README.md [hunk 3]
```diff
@@ -1,27 +1,79 @@
...
+```
+
+6. Run the frontend in a seperate terminal from the backend
+
+```bash
+    yarn start consultation-portal
+```
+
+7. Navigate to [localhost:4200/consultation-portal](http://localhost:4200/consultation-portal)
+
```
Comment (diljath) [inline apps/consultation-portal/README.md:63]: ætti þetta ekki að vera /samradsgatt ?

**Review by diljath — APPROVED** [review]

**Review by eirikurn — APPROVED** [review]
LGTM

**Pull request merged**